server/bin/bind-performance.rb

```diff
@@ -26,9 +29,9 @@ def time_rand from = 0.0, to = Time.now
   Time.at(from + rand * (to.to_f - from.to_f))
 end
 
-cp = Candlepin.new(ADMIN_USERNAME, ADMIN_PASSWORD, nil, nil, HOST, PORT)
+@cp = Candlepin.new(ADMIN_USERNAME, ADMIN_PASSWORD, nil, nil, HOST, PORT)
```

Comment:
Why change this to an instance variable? Seems like it shouldn't work, and definitely doesn't make sense semantically.